server/app/routes/user/index.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+const User = require('../../../db/models/user');
+// const Cart = require('../../../db/models/cart');
+
```

Comment:
probably a good place to have a `router.param`, e.g.:

```js
const userId;
router.param('userId', function (request, response, next, id) {
	request.body.userId = parseInt(id, 10);
	next();
});

router.use('/', (request, response, next) => userId = request.body.userId
```

But probably want to check if this is right w/Gabe(?)